content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -271,6 +271,13 @@ const Exhibition: FunctionComponent<Props> = ({
     />
   );
 
+  const exhibitionFormat =
```

Comment:
I don't think this needs to be in the form of an array with an object with a `text` key? It could just return the individual strings ('Exhibition' or `exhibition.format?.title`)?

Reply:
sorry just realised that's what Raph said

Reply:
Thanks both! Good spot. This is of course correct - about to push this change. 😄